lib/mayaUsd/ufe/UsdUndoRenameCommand.h

```diff
@@ -59,12 +59,12 @@ class MAYAUSD_CORE_PUBLIC UsdUndoRenameCommand : public Ufe::UndoableCommand
 	bool renameRedo();
 	bool renameUndo();
 
-	UsdStageWeakPtr fStage;
-	SdfLayerHandle fLayer;
-    UsdSceneItem::Ptr fUfeSrcItem;
-	SdfPath fUsdSrcPath;
-	UsdSceneItem::Ptr fUfeDstItem;
-	SdfPath fUsdDstPath;
+	UsdStageWeakPtr m_fStage;
+	SdfLayerHandle m_fLayer;
+    UsdSceneItem::Ptr m_fUfeSrcItem;
+	SdfPath m_fUsdSrcPath;
+	UsdSceneItem::Ptr m_fUfeDstItem;
+	SdfPath m_fUsdDstPath;
```

Comment:
Following the C++ guideline, member variables need to be prefixed with `m_ `

Reply:
Actually, we use `_` prefix - see https://github.com/Autodesk/maya-usd/blob/dev/doc/codingGuidelines.md#naming-file-type-variable-constant-function-namespace-macro-template-parameters-schema-names

New modules should start using these guidelines, but we still have to plan how to adapt the existing code.

Reply:
My bad, I was sill thinking about "m_" :)   Fixed in 54166d2b6bc